### PR TITLE
CIFAR-100 Efficient Net v2 example

### PR DESCRIPTION
**Pull request by vicloginov**

### CODE REVIEW

File: models/ai87net-effnetv2.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+###################################################################################################
+#
+# Copyright (C) Maxim Integrated Products, Inc. All Rights Reserved.
```
Comment (aniktash) [inline models/ai87net-effnetv2.py:3]: Date for copyright

File: models/ai87net-effnetv2.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+def ai87effnetv2(pretrained=False, **kwargs):
+    """
+    Constructs EfficientNet v2 model.
+    """
+    assert not pretrained
+    return AI87EfficientNetV2(**kwargs)
+
+models = [
+    {
+        'name': 'ai87effnetv2',
```
Comment (aniktash) [inline models/ai87net-effnetv2.py:88]: YAML in synthesis refers to arch as ai85effnetv2_n?

Comment (vicloginov) [reply models/ai87net-effnetv2.py:88]: I will fix it

File: models/ai87net-effnetv2.py (hunk 1 above)
Comment (vicloginov) [reply models/ai87net-effnetv2.py:3]: Sure

File: models/ai87net-effnetv2.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            bias=False,
+            **kwargs
+    ):
+        super().__init__()
+
+        # Stem Layer
+        self.conv_stem = ai8x.FusedMaxPoolConv2dBNReLU(num_channels, 32, 3, pool_size=2, pool_stride=2, stride=1, batchnorm='Affine', padding=1, bias=bias,
+                                            eps=1e-03, momentum=0.01, **kwargs)
+        # Series of MBConv blocks
+        self.mb_conv1 = ai8x.MBConvBlock(32, 16, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=1,
```
Comment (MaximGorkem) [inline models/ai87net-effnetv2.py:33]: padding argument is not used in MBConvBlock, should we consider removing it from init not to create confusion?

**Review by MaximGorkem — APPROVED** [review]
Great implementation. It is the closest form of effnet_v2 with the HW blocks we have. Thanks

File: models/ai87net-effnetv2.py (hunk 3 above)
Comment (vicloginov) [reply models/ai87net-effnetv2.py:33]: Sure

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**